src/components/Contacts/ContactFilters/ContactFilters.tsx

```diff
@@ -0,0 +1,35 @@
+import { colors } from '@material-ui/core';
+import React from 'react';
+import { useContactFiltersLazyQuery } from '../../../../pages/accountLists/[accountListId]/ContactFilters.generated';
+
+interface Props {
+  accountListId: string;
+}
+
+export const ContactFilters: React.FC<Props> = ({ accountListId }: Props) => {
+  const [
+    loadContactFilters,
+    { data, loading, error },
+  ] = useContactFiltersLazyQuery({
+    variables: { accountListId: accountListId as string },
```

Comment:
This is already a string right?
```suggestion
    variables: { accountListId: accountListId },
```